File: runelite-client/src/main/java/net/runelite/client/plugins/microbot/vorkath/STAFF.java
package net.runelite.client.plugins.microbot.vorkath;

import lombok.Getter;
import lombok.RequiredArgsConstructor;

@Getter
@RequiredArgsConstructor
public enum STAFF {
    CAST("Cast");

    private final String staffName;

    @Override
    public String toString() {
        return staffName;
    }

}

File: runelite-client/src/main/java/net/runelite/client/plugins/jrPlugins/autoVorkath/STAFF.kt
package net.runelite.client.plugins.jrPlugins.autoVorkath

import lombok.Getter
import lombok.RequiredArgsConstructor

@Getter
@RequiredArgsConstructor
enum class STAFF(val staffName: String) {
    CAST("Cast"),
    SLAYER_STAFF("Slayer's Staff"),
    SLAYER_STAFF_E("Slayer's Staff (e)");

    override fun toString(): String { return staffName }
}
